packages/documentation-spa/docs/onboarding-guide/onboarding-guide.md

```diff
@@ -0,0 +1,283 @@
+---
+id: onboarding-guide
+title: One Platform Onboarding Guide
+sidebar_label: Onboarding Guide
+slug: /onboarding-guide
+---
+* * * *
+
+
+## **PURPOSE**
+The purpose of this Guide is to help new developers to get started with One Platform. This will cover various aspects such as -  Overview of One Platform, Its Architecture, Services etc. So let’s get started. Happy Reading!!
+
+
+## **OVERVIEW**
+ One Platform is a single place solution for teams within Red Hat to deploy their applications, websites and Microservices. The Stakeholders can easily deploy their applications without even being bothered about building and managing the Infrastructure hussles.
```

Comment:
```suggestion
 One Platform is a single stop solution for teams within Red Hat to deploy their applications, websites and Microservices. The Stakeholders can easily deploy their applications without even being bothered about building and managing the Infrastructure hussles.
```

Reply:
@riginoommen can we add something better here?